docs/product/sentry-basics/integrate-frontend/configure-scms.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: Enable Suspect Commits & Stack Trace Linking
 sidebar_order: 5
-description: "Learn how Sentry uses commit metadata from your source code repositories to help you resolve your issues faster."
+description: "Learn how to integrate your source code with Sentry, so that Sentry can track commits and surface suspect commits and susgested assignees to help you resolve issues faster."
```

Comment:
```suggestion
description: "Learn how to integrate your source code with Sentry, so that Sentry can track commits and surface suspect commits and suggested assignees to help you resolve issues faster."
```